tools/import/importservice.js

```diff
@@ -118,16 +118,35 @@ export default class ImportService {
     localStorage.setItem(STORAGE_JOBS, JSON.stringify([]));
   }
 
-  async startJob({ urls = [], options = {}, importScript }) {
+  /**
+   * Start a bulk importer job.
+   * @param urls An array of URLs to import.
+   * @param options The options to use during the import.
+   * @param customHeaders A string, in JSON object format, specifying the headers to use during
+   *                      the import.
+   * @param importScript The import script to upload and use for the import.
+   * @returns {Promise<*|{}>}
+   */
+  async startJob({
+    urls = [], options = {}, customHeaders = '', importScript,
+  }) {
     if (!this.apiKey) {
       throw new Error('API key is required');
     }
-    const body = importScript ? { urls, options, importScript } : { urls, options };
     try {
+      const formData = new FormData();
+      formData.append('urls', JSON.stringify(urls));
+      formData.append('options', JSON.stringify(options));
+      formData.append('customHeaders', customHeaders);
+      formData.append('importScript', importScript);
+
+      const authHeaders = this.#getAuthHeaders();
+      delete authHeaders['Content-Type'];
```

Comment:
I am still seeing application/json in the request when using this approach.

Reply:
Bruce and I saw it in the Response header, but not in the Request header which defaulted correctly.